samples/popup/popup.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Popup Demo</title>
+    <meta name="description" content="Popup demo" />
+    <script src="https://aframe.io/releases/1.2.0/aframe.min.js"></script>
+    <script src="prism.js"></script>
+  </head>
+  <body>
+    <a-scene background="color: #ECECEC" cursor="rayOrigin: mouse">
+      <a-entity
+        id="primaryCamera"
+        position="0 0 9"
+        rotation="0 0 0"
+        camera
+        look-controls
+        wasd-controls
+      ></a-entity>
+      <a-entity id="readRoom" position="0 -100 0">
+        <a-entity
+          id="blackboard"
+          info-panel
```

Comment:
Is `info-panel` an Aframe thingy? Or just leftover from something?

Reply:
leftover, will remove